docs/api-reference/event-manager.md

```diff
@@ -2,10 +2,15 @@
 
 Provides a unified API for subscribing to events about both basic input events (e.g. 'mousemove', 'touchstart', 'wheel') and gestural input (e.g. 'click', 'tap', 'panstart').
 
+Delegates gesture-related event registration and handling to Hammer.js. Includes shims for handling event not supported by Hammer.js, such as keyboard input, mouse move, and wheel input.
+
 ## Usage
 
 TBA
 
+(What is the intent of this section? Install/import instructions?)
```

Comment:
It is intended for a couple of examples of how to use this class. 

Create an event manager
```js
const manager = new EventManager({
```

Register events
...